addon/components/nrg-lightbox-thumbnail/template.hbs

```diff
@@ -1 +1,2 @@
 <img role="button" class="ui image photo-thumbnail" src={{photo.url}} alt={{photo.altText}} {{action "openLightbox"}}>
+<div style="display:none" id="thumb-{{uuid}}">{{yield}}</div>
```

Comment:
Can you copy the component from our internal app to allow for better expansion and flexibility for other use cases?